packages/schemas/tables/migrations.sql

```diff
@@ -0,0 +1,10 @@
+create table _migrations (
```

Comment:
nit: Just asking, maybe single form is OK? Say `_migration`

Reply:
But all other table names are in plural.

Reply:
ok, np